### ISSUE TITLE
[WIP] Add networking features to Lib5K

### ISSUE THREAD

**Issue by ewpratten** [frc5024/uBase#15]
Desc TODO

**Comment by ewpratten** [comment]
I have added a network port management system. Should test it all out before saying it is done

**Comment by ewpratten** [comment]
Waiting on WPILib v2020 before I add automatic Limelight portforwarding

**Comment by ewpratten** [comment]
Merged in #17 

Will open another issue for auto-PortForwarding Limelights
